### PR TITLE
Heading visible while loading

### LINKED ISSUE #16947: Heading still visible when the appsWhatIsIt is loading
Screenshot from 2020-03-20 16-31-00

### PR DESCRIPTION
**Pull request by ritwizsinha**
Screenshot from 2020-03-20 16-37-08

Closes #16947

### CODE REVIEW

**Comment by ritwizsinha** [comment]
@gabriellsh please review

**Comment by gabriellsh** [comment]
Why do you think the header shouldn't be visible when loading the page?

**Comment by ritwizsinha** [comment]
@gabriellsh this is because the header "Apps what are they " is the header only for when the apps framework is not enabled and the description of apps is given. When apps framework is enabled then the page transitions to the apps list page which is in no way related to the header. Thus while loading too there is no need for the header .

**Comment by gabriellsh** [comment]
https://github.com/RocketChat/Rocket.Chat/blob/f32703f59b1e28bc11394fe570525aa1c18359b9/app/apps/client/routes.js#L11

It's not supposed to render that template before it checks if apps are enabled or not. Are you sure it's rendering the page with apps enabled?

File: app/apps/client/admin/appWhatIsIt.html [hunk 1]
```diff
@@ -1,15 +1,19 @@
 <template name="appWhatIsIt">
 	<section class="page-container page-home page-static content-background-color">
-		{{> header sectionName="Apps_WhatIsIt" hideHelp=true fixedHeight=true}}
+		{{#if isLoading}}
```
Comment (gabriellsh) [inline app/apps/client/admin/appWhatIsIt.html]: The `#if` should be inside the content div, there is always content in the page.

**Comment by gabriellsh** [comment]
This page was refactored recently, so this is no longer an issue.

**Pull request closed without merging**